packages/shared/routes/login/views/EnterPinView.svelte

```diff
@@ -97,19 +97,19 @@
             const profile = $activeProfile
             isBusy = true
 
+            let clientOptions = $activeProfile.clientOptions
+            if (clientOptions?.nodes?.length < 1) {
+                clientOptions = getDefaultClientOptions($activeProfile.networkProtocol, $activeProfile.networkType)
+            }
+
```

Comment:
What cases are there where the length of the nodes array is less than 1?

Reply:
In best case it wouldn't, but the other piece of code removed was causing the nodes to be deleted when you visit the network config page

Reply:
This is to recover those profiles that where corrupted. I guess it's only a problem in the beta

Reply:
From issue #4110 
`Some profiles are no longer able to login because they don't have any nodes in the persisted client options. The client options where passed into the initialise profile manager every-time because wallet.rs corrupted them after a breaking change. But somehow the node is being deleted from the persisted storage.`

Not sure how wallet.rs is deleting them though